### DISCUSSION TITLE
How to adjust plot area size

### DISCUSSION THREAD

**Discussion by brianpenghe** [GreenleafLab/ArchR#1112] [Questions / Documentation] [answered]
I have more than 80 clusters in my atlas.
When I plot using `plotEmbedding` I get this:
![image](https://user-images.githubusercontent.com/4110443/139114378-9fda66f8-addb-425f-8261-952eff3ea4ce.png)

Is there a way to show a bigger plot area?

Thanks!

**Comment by rcorces** [comment, marked as answer]
I'm not sure what you mean. Plot size is controlled when the plot is saved to a file.
Also, if you look at the `...` pass-through arguments (at the bottom of the function params), `plotEmbedding()` allows you to pass arguments through to `ggPoint()`.

**Reply by brianpenghe** [reply]
Thanks! I managed to ask ggplot to adjust the layout.
